rust/.config/nextest.toml

```diff
@@ -6,7 +6,7 @@ uses-engine = { max-threads = 4 }
 after-engine = { max-threads = 12 }
 
 [profile.default]
-slow-timeout = { period = "30s", terminate-after = 1 }
+slow-timeout = { period = "90s", terminate-after = 1 }
```

Comment:
Is this intentional? If so, we probably want to bump up the `ci` profile timeout (below) also.

Reply:
It is intentional. My assumption is that it is just slow for me in NZ and that in CI and for everyone living in the civilised world, it completes in time.